.ado/pipelines/templates/steps-frontdoor-traffic-switch.yaml

```
@@ -95,47 +102,49 @@ steps:
           $queryOptionEnabledOnly = "?enabledState=='Enabled'" # If the switch to remove disabled backends is true, we only fetch the currently enabled ones and thereby not re-adding the disabled ones
         }
 
-        # Fetch existing backends in the pool from Front Door. We need the attributes "address", "weight" and "enabledState"
-        $backends = $(az network front-door backend-pool backend list `
-                        --front-door-name $(frontdoor_name) `
-                        --pool-name $pool `
-                        --resource-group $(global_resource_group_name) `
-                        --query "[$queryOptionEnabledOnly].{address:address,weight:weight,enabled:enabledState}")
+        # Fetch existing backends in the pool from Front Door
+        $backends = az afd origin list `
+                    --profile-name $(frontdoor_name) `
+                    --origin-group-name $pool -g $(global_resource_group_name) `
+                    --query "[$queryOptionEnabledOnly]" | ConvertFrom-Json -NoEnumerate
```

```suggestion
        $backends = az afd origin list `
                    --profile-name $(frontdoor_name) `
                    --origin-group-name $pool `
                    --resource-group $(global_resource_group_name) `
                    --query "[$queryOptionEnabledOnly]" | ConvertFrom-Json -NoEnumerate
```

for consistency.. ;)